### DISCUSSION TITLE
SmartAutoClicker-1.5.1

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#140] [Announcements]
# New Feature:

## Chinese translation
Smart AutoClicker is now fully translated in chinese. Thanks to @jiongjiongJOJO !

# Bug fixes:

## Crash while creating a backup of a scenario
This was only occurring in release version of the application. It is now fixed.

 

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/1.5.1'>SmartAutoClicker-1.5.1</a>.</em>

**Comment by zguinga23** [comment]
Hello Nain57. Can you add a click option on the color change of a pixel, or allow to shrink the image capture as much as possible, in order to gain more execution speed?

**Comment by testu5er1** [comment]
Hello Nain57, Thanks for creating this app. It is so simple and effective for most of the games. A small request here. Is it possible to request an option to adjust the scan by a fixed timer to conserve battery life? 

For example, run the scan after a period time/second instead of every millisecond. This is because the current battery consumption is quite high, and if there is an option to adjust the scan timer, it would be great.

Thanks.
